omod/src/main/java/org/openmrs/module/webservices/rest/resource/ItemStockDetailResource.java

```diff
@@ -11,56 +9,79 @@
 import org.openmrs.module.openhmis.inventory.api.model.ItemStockDetail;
 import org.openmrs.module.openhmis.inventory.api.model.Stockroom;
 import org.openmrs.module.openhmis.inventory.web.ModuleRestConstants;
+import org.openmrs.module.webservices.rest.SimpleObject;
 import org.openmrs.module.webservices.rest.web.RequestContext;
 import org.openmrs.module.webservices.rest.web.annotation.Resource;
 import org.openmrs.module.webservices.rest.web.representation.Representation;
 import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
 import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
 
-@Resource(name=ModuleRestConstants.ITEM_STOCK_DETAIL_RESOURCE, supportedClass = ItemStockDetail.class,
-		supportedOpenmrsVersions={"1.9.*", "1.10.*", "1.11.*" })
-public class ItemStockDetailResource extends ItemStockDetailBaseResource<ItemStockDetail> {
+import java.util.List;
 
+@Resource(name = ModuleRestConstants.ITEM_STOCK_DETAIL_RESOURCE, supportedClass = ItemStockDetail.class,
+        supportedOpenmrsVersions = { "1.9.*", "1.10.*", "1.11.*" })
+public class ItemStockDetailResource extends ItemStockDetailBaseResource<ItemStockDetail> {
+	
 	private IStockroomDataService stockroomDataService;
 	private IItemStockDetailDataService itemStockDetailDataService;
-
+	
 	public ItemStockDetailResource() {
 		this.stockroomDataService = Context.getService(IStockroomDataService.class);
 		this.itemStockDetailDataService = Context.getService(IItemStockDetailDataService.class);
 	}
-
+	
 	@Override
 	public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
-		DelegatingResourceDescription description =  super.getRepresentationDescription(rep);
+		DelegatingResourceDescription description = super.getRepresentationDescription(rep);
 		description.addProperty("stockroom", Representation.DEFAULT);
 		return description;
 	}
-
+	
 	@Override
 	protected PageableResult doSearch(RequestContext context) {
```

Comment:
This is almost identical to the getAll function below. Are they both needed? If so, reuse the code rather than copy/paste.